### PR TITLE
Feat: achievements empty states

### PR DESCRIPTION
**Pull request by LucasFalkowsky**
closes https://github.com/corona-school/project-user/issues/1067

### What was done:

- empty state component was introduced

### CODE REVIEW

File: src/components/achievements/EmptyStateContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+const EmptyStateContainer = ({ achievementState }: EmptyStateContainerProps) => {
+    const { t } = useTranslation();
+    switch (achievementState) {
+        case Achievement_State.Completed:
+            return (
+                <VStack width="100%" display="flex" flexDirection="column" alignItems="center" space="4px">
+                    <EmptyStateCompleted />
+                    <Text fontWeight={600} fontSize="14px" lineHeight="17px" textAlign="center">
```
Comment (LomyW) [inline src/components/achievements/EmptyStateContainer.tsx:17]: can we please use the pre-defined fontSizes like 'xs', 'sm' etc.?

File: src/components/achievements/EmptyStateContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                        {t('achievement.empty_state.Completed.title')}
+                    </Text>
+                    <Text fontWeight={400} fontSize="10px" lineHeight="12px" textAlign="center">
```
Comment (LomyW) [inline src/components/achievements/EmptyStateContainer.tsx:20]: also here? I think 10px is little bit too small

Comment (LomyW) [inline src/components/achievements/EmptyStateContainer.tsx:20]: also here, think 10px is little bit to small

File: src/components/achievements/EmptyStateContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                        {t('achievement.empty_state.Completed.description')}
+                    </Text>
```
Comment (LomyW) [inline src/components/achievements/EmptyStateContainer.tsx:22]: maybe something like that? (that it looks similar to the other empty states in the user app i.e. `AppointmentEmptyState``)
```suggestion
                     <Heading>{t('achievement.empty_state.Completed.title')}</Heading>
                    <Text maxW="700" textAlign="center" py="3">
                        {t('achievement.empty_state.Completed.description')}
                    </Text>
```

File: src/widgets/AchievementProgress.tsx [hunk 4]
```diff
@@ -191,46 +192,58 @@ const AchievementProgress: React.FC<AchievementProgressProps> = ({ achievements,
                 {states.map((key) => (
                     <VStack key={key} space={3} marginTop={10}>
                         <ProgressCollapsableHeadline achievementState={key} onClick={() => handleOnClick(undefined, key)} />
```
Comment (LomyW) [inline src/widgets/AchievementProgress.tsx:194]: is the empty state also collapsable?

**Review by LomyW — CHANGES_REQUESTED** [review]
some notes ✨

File: src/components/achievements/EmptyStateContainer.tsx (hunk 2 above)
Comment (LucasFalkowsky) [reply src/components/achievements/EmptyStateContainer.tsx:20]: It is exactly what the design sais. Thats why I used these values.
I changed them to the presets though.

File: src/components/achievements/EmptyStateContainer.tsx (hunk 3 above)
Comment (LucasFalkowsky) [reply src/components/achievements/EmptyStateContainer.tsx:22]: But this is not as it was done in the design.

**Review by LomyW — APPROVED** [review]
Before merging please resolve the conflict.
Looks good to me. 🎉

**Pull request merged**